README.md

```diff
@@ -31,9 +31,11 @@ that apply to that project.
 *   [TypeScript Style Guide][ts]
 *   [Vim script Style Guide][vim]
 
-This project also contains [cpplint][cpplint], a tool to assist with style guide
-compliance, and [google-c-style.el][emacs], an Emacs settings file for Google
-style.
+This project also contains [google-c-style.el][emacs], an Emacs settings file
+for Google style.
+
+We used to host the [cpplint tool](https://github.com/cpplint/cpplint), but it
```

Comment:
This markup is slightly misleading, it could be read as if Google hosted https://github.com/cpplint/cpplint. I would suggest:

```
We used to host the google internal cpplint tool, but won't be pushing updates. Open source forks such as [cpplint on PyPi](https://github.com/cpplint/cpplint) exist.
```